src/eolab/georastertools/processing/vector.py

```diff
@@ -66,25 +66,26 @@ def filter(geoms: Union[gpd.GeoDataFrame, Path, str], raster: Union[Path, str],
     geoms_crs = _get_geoms_crs(geometries)
 
     file = raster.as_posix() if isinstance(raster, Path) else raster
-    with rasterio.open(file) as dataset:
-        l, b, r, t = dataset.bounds
-        px, py = ([l, l, r, r], [b, t, t, b])
 
-        if(geoms_crs != dataset.crs):
-            px, py = warp.transform(dataset.crs, geoms_crs, [l, l, r, r], [b, t, t, b])
+    dataset = rasterio.open(file)
```

Comment:
Why did you replace with context manager with a standard open call in many places in the code?